src/main/java/org/terasology/genome/breed/BiodiversityGenerator.java

```diff
@@ -38,6 +36,8 @@
     private UniformNoiseAlpha uniformNoise = new UniformNoiseAlpha(IdentityAlphaFunction.singleton());
 
     /**
+     * Parametrized constructor.
```

Comment:
Not super useful info being added here. Might be better with something like "Constructor preparing a range of noise and probabilities to be used by this generator."

I was going to say "For the world" but oddly it sounds like there could be multiple instances of this generator class.